### PR TITLE
refactor: use `message.send()` directly, move message to container

### PR DESCRIPTION
**Pull request by sidvishnoi**
## Context

Closes https://github.com/interledger/web-monetization-extension/issues/499

## Changes proposed in this pull request

- Move `MessageManager` instance into container/context so we don't have to import `browser` at top-level (makes browser easily replaceable for testing)
- Reduce indirection when sending messages, and simplify call signature - aligning it more with how we use `SendToPopup`

### CODE REVIEW

File: src/content/services/monetizationTagManager.ts [hunk 1]
```diff
@@ -94,7 +89,7 @@ export class MonetizationTagManager extends EventEmitter {
   }
 
   private async resumeAllMonetization() {
-    const response = await isWMEnabled()
+    const response = await this.message.send('IS_WM_ENABLED', undefined)
```
Comment (raducristianpopa) [inline src/content/services/monetizationTagManager.ts:92]: Look if we can avoid passing undefined as the second parameter (the payload).

File: src/popup/components/ErrorKeyRevoked.tsx [hunk 2]
```diff
@@ -11,8 +11,8 @@ import type { Response } from '@/shared/messages'
 
 interface Props {
   info: Pick<PopupStore, 'publicKey' | 'walletAddress'>
-  disconnectWallet: () => Promise<Response>
-  reconnectWallet: () => Promise<Response>
+  disconnectWallet: () => Promise<Response<void>>
```
Comment (raducristianpopa) [inline src/popup/components/ErrorKeyRevoked.tsx:14]: Check if we can avoid passing void to the `Response` type.

File: src/popup/components/OutOfFunds.tsx [hunk 3]
```diff
@@ -68,7 +68,7 @@ interface AddFundsProps {
   info: Pick<WalletAddress, 'id' | 'assetCode' | 'assetScale'>
   recurring: boolean
   defaultAmount: AmountValue
-  requestAddFunds: (details: AddFundsPayload) => Promise<Response>
+  requestAddFunds: (details: AddFundsPayload) => Promise<Response<void>>
```
Comment (raducristianpopa) [inline src/popup/components/OutOfFunds.tsx:71]: Ditto above.

File: src/content/services/monetizationTagManager.ts (hunk 1 above)
Comment (sidvishnoi) [reply src/content/services/monetizationTagManager.ts]: Done in https://github.com/interledger/web-monetization-extension/pull/534/commits/2c178a023bf5c175caf12b548b4cd7c2e839bf63.

**Review by raducristianpopa — APPROVED** [review]

**Pull request merged**